package.yaml

```diff
@@ -26,6 +26,27 @@ library:
     - Hpack.Config
     - Hpack.Run
     - Hpack.Yaml
+  # You can obtain this list "handy" with:
+  # grep 'LANGUAGE' src/**/*.hs|perl -pe 's/^.*LANGUAGE /    - /; s/ #-\}//'|sort -u
+  other-extensions:
+    - CPP
+    - DefaultSignatures
+    - DeriveDataTypeable
+    - DeriveFoldable
+    - DeriveFunctor
+    - DeriveGeneric
+    - DeriveTraversable
+    - FlexibleContexts
+    - FlexibleInstances
+    - OverloadedStrings
+    - PolyKinds
+    - QuasiQuotes
+    - RecordWildCards
+    - ScopedTypeVariables
+    - TypeFamilies
+    - TypeOperators
+    - UndecidableInstances
+    - ViewPatterns
```

Comment:
Unless somebody can point out how this information is essential, I'm not eager to maintain this list manually.

As I understand it, it's possible to derive this list of extensions by looking at the source files.  If we want to have this information in the cabal file for completeness, I guess `hpack` could infer it by looking at the source files.